### DISCUSSION TITLE
BIG HEADS SVG.

### DISCUSSION THREAD

**Discussion by utsav-devadiga** [dicebear/dicebear#201] [Ideas]
integration of the bigheads into this platform will be great

https://bigheads.io/

**Comment by FlorianKoerner** [comment]
I had already talked to Robert (@RobertBroersma), the developer behind big heads, a few months ago. Unfortunately he doesn't have the source files for the avatars anymore.

At the same time, I'm not sure if he's also the designer or just the developer.

We need the designer's approval and the corresponding source files for an implementation.
